src/ir/comp.rs

```diff
@@ -1517,6 +1517,10 @@ impl CompInfo {
             }) {
                 info!("Found a struct that was defined within `#pragma packed(...)`");
                 return true;
+            } else if self.has_own_virtual_method {
```

Comment:
It passes either way, so I guess it doesn't really matter.  A class without data members won't ever pass that first condition.  I can reword it if you like.

Reply:
No, that's fine, I was wondering if this would allow catching stuff where we don't know the layout of the members because libclang doesn't expose it, yet we know we have a vtable. Anyhow this is fine.